src/tir/transforms/lower_sparse_tir.cc

```diff
@@ -260,29 +322,146 @@ class IndexTransformer : public StmtExprMutator {
   }
 
   PrimExpr VisitExpr_(const SparseBufferLoadNode* load) final {
+    buffer_read_.insert(load->buffer.get());
     PrimExpr lowered_indices = LowerIndices(load->buffer, load->indices);
     return BufferLoad(load->buffer->data, {std::move(lowered_indices)});
   }
 
   Stmt VisitStmt_(const SparseBufferStoreNode* store) final {
+    buffer_write_.insert(store->buffer.get());
     PrimExpr value = ExprMutator::VisitExpr(store->value);
     PrimExpr lowered_indices = LowerIndices(store->buffer, store->indices);
     return BufferStore(store->buffer->data, std::move(value), {std::move(lowered_indices)});
   }
 
-  BufferAccessMap buffer_access_map_;
-  DependencyMap dependency_map_;
+  Stmt VisitStmt_(const SparseBlockNode* sp_block) {
```

Comment:
where is the block isolation generated?

Reply:
Haven't generate separating blocks yet. This is because I haven't figured out the isolation rule very well. Let's do it in future PRs.